ppr-ui/src/composables/mhrRegistration/home-owner.ts

```diff
@@ -284,22 +292,46 @@ export function useHomeOwners (isPerson: boolean = false, isEditMode: boolean =
     return { updatedGroups, updatedFractionalData }
   }
 
+  /**
+  * Both getMhrRegistrationHomeOwnerGroups and getMhrRegistrationHomeOwners are initialized with a Group value of "1"
+  * This logic checks the following and produces the heading based on whether it is a group which contains owners
+  * or owners without grous. * Groups = getMhrRegistrationHomeOwnerGroups.value.length === 0 //not sure if this
+  * will ever happen.
+  * Owners = (getMhrRegistrationHomeOwnerGroups.value.length === 1 && !getMhrRegistrationHomeOwners.value.address) &&
+    getMhrRegistrationHomeOwnerGroups.value[0].owners.length <= 1 &&
+     getMhrRegistrationHomeOwnerGroups.value[0].interestNumerator === null
+      (The above code simply checks for the presence of a group with 1 group and one owner.
+      When all owners are deleted 1 owner still exists without
+      and address and the interestinumerator will be blank... )
+  */
+  const showGroupHeading = (): Boolean => {
```

Comment:
Not sure we need this? Yes, the groups are initialized with '1', but the initial value for `showGroups` is false. It will be set to true once the user decides to add a Group. The only time the groups length will be 0 is when we delete all groups and therefore reset `showGroups` to false.

Reply:
The reason I put in this logic, is because is because when a person was deleted, it added a  group heading and displayed Tenants in Common.. This logic fixed that issue...

eg.  
![image](https://user-images.githubusercontent.com/107890689/191144402-e37c49dc-6f63-42d8-b9d4-95598a7c82bf.png)

Reply:
Is there a way to simplify it? Also, `getMhrRegistrationHomeOwners` is an array, so this `getMhrRegistrationHomeOwners.value.address` will not work...

Reply:
Worked with Dima to rework logic.  fixed.